gaatha/settings.py

```diff
@@ -0,0 +1,175 @@
+"""
+Django settings for gaatha project.
+
+Generated by 'django-admin startproject' using Django 4.1.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+import os
+from pathlib import Path
+import environ
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+env = environ.Env(
+    DEBUG=(bool, True),
+    SECRET_KEY=(str, 'django-insecure-pm5c((7v2kh7btd(o%2exi4+6_ui1pso-6naxq-6zuaxs7sq+%'),
```

Comment:
Let's never add a secret key here
```suggestion
    DJANGO_SECRET_KEY=str,
```